From 79658079a0113d14f7c74cfc411860ddbc625a74 Mon Sep 17 00:00:00 2001
From: Yanjiang Jin <yanjiang.jin@windriver.com>
Date: Thu, 28 Jul 2016 10:59:56 +0800
Subject: [PATCH 828/838] fsl_dce: set two symbols to maybe_unused

This is to avoid the below compile warnings:

In file included from drivers/staging/fsl_dce/tests/performance_simple/dce_sf_perf_simple.c:62:0:
drivers/staging/fsl_dce/tests/performance_simple/../common/paper6_12K_compressed.gz.h:1:22: warning: 'paper6_12K_compressed_gz' defined but not used [-Wunused-variable]
 static unsigned char paper6_12K_compressed_gz[] = {
                      ^
drivers/staging/fsl_dce/tests/performance_simple/../common/paper6_12K_compressed.gz.h:1029:21: warning: 'paper6_12K_compressed_gz_len' defined but not used [-Wunused-variable]
 static unsigned int paper6_12K_compressed_gz_len = 12306;
                     ^

Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/staging/fsl_dce/tests/common/paper6_12K_compressed.gz.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl_dce/tests/common/paper6_12K_compressed.gz.h b/drivers/staging/fsl_dce/tests/common/paper6_12K_compressed.gz.h
index f0884c2..e9a9146 100644
--- a/drivers/staging/fsl_dce/tests/common/paper6_12K_compressed.gz.h
+++ b/drivers/staging/fsl_dce/tests/common/paper6_12K_compressed.gz.h
@@ -1,4 +1,4 @@
-static unsigned char paper6_12K_compressed_gz[] = {
+static unsigned char __maybe_unused paper6_12K_compressed_gz[] = {
 	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x03, 0xdd, 0x7d,
 	0x6b, 0x73, 0xdb, 0x46, 0x96, 0xe8, 0x67, 0xf5, 0xaf, 0xe8, 0xe5, 0x20,
 	0x6b, 0xa9, 0x4c, 0x32, 0x24, 0xf5, 0xf2, 0x23, 0xce, 0x94, 0xa2, 0x38,
@@ -1026,4 +1026,4 @@ static unsigned char paper6_12K_compressed_gz[] = {
 	0x8c, 0xfc, 0x20, 0x07, 0xf8, 0x3f, 0xc1, 0xa6, 0xfe, 0x2f, 0x79, 0xdc,
 	0xcf, 0x16, 0x75, 0x86, 0x00, 0x00
 };
-static unsigned int paper6_12K_compressed_gz_len = 12306;
+static unsigned int __maybe_unused paper6_12K_compressed_gz_len = 12306;
-- 
2.0.2

